templates/example_cell_clustering.ipynb

```diff
@@ -39,12 +39,15 @@
     "\n",
```

Comment:
<p>I don't think the note about NA/NaN/Inf is needed here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>